tests/test_integration_real_llm.py

```
@@ -0,0 +1,127 @@
+import pytest
+import asyncio
+from unittest.mock import patch
+
+from mad_spark_alt.core import SimpleQADIOrchestrator, setup_llm_providers
+
+
+class TestRealLLMIntegration:
+    """Integration tests that use real LLM calls to validate the system."""
+
+    @pytest.fixture(scope="class")
+    async def llm_setup(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `llm_setup`

Add return type annotation: `None`

(ANN201)